frontend/coprs_frontend/coprs/templates/api.html

```diff
@@ -45,6 +45,17 @@ <h2>API Token</h2>
         <input type="button" value="Generate a new token" />
     </a>
     {% else %}
+    <pre style="font-size:120%">
+[copr-cli]
+login = VALUE-IS-HIDDEN
```

Comment:
Nice idea, thanks!  Can you `s/VALUE-IS-HIDDEN/LOGIN_TO_UNHIDE/`?  I observed the page after the change, and the info that you have to log-in is pretty invisible now.

Reply:
UNHIDE is not a word. I think. I used REVEAL. 
Updated.